.github/workflows/publish-docs.yml

```diff
@@ -0,0 +1,23 @@
+name: 'Publish Storybook'
+on:
+  push:
+    branches:
+      - develop
```

Comment:
Build and publish docs on pushes to `develop`. No reason to tie Storybook updates with tagged releases.